activate.sh

```diff
@@ -58,3 +36,4 @@ echo "id: admin" > /etc/salt/minion.d/minion_id.conf
 echo "grains_refresh_every: 10" > /etc/salt/minion.d/grains_refresh.conf
 
 systemctl enable salt-minion
+systemctl enable admin-node-setup
```

Comment:
This won't work right on cloud-init based deployments (and the salt-minion one is due to be removed). This needs to be added to YaST and our cloud-init module.

That said, we should leave it here for now - because we have an ordering problem to get all 3 changes landed! Leaving it here will mean we don't break things while we await the other 2 changes.